src/@test/utilsTest/FinalScore.test.ts

```diff
@@ -0,0 +1,29 @@
+import { FinalScore } from '../../utils/FinalScore';
+
+describe('FinalScore', () => {
+  describe('calculateTotalScore', () => {
+    test('should return the correct total score when all elements in the array are valid scores', () => {
```

Comment:
Você seguiu o padrão aqui, mas nos outros de forma diferente. Você precisa estabelecer um único em todo o projeto ao invés de ter vários ao mesmo tempo.

"It should be able to..."
"It should not be able to..."